docs/appendices/release-notes/unreleased.rst

```diff
@@ -72,6 +72,9 @@ Changes
 - Users can now read tables within the ``pg_catalog`` schema without explicit
   ``DQL`` permission. They will only see records the user has privileges on.
 
+- Added support for the ``START TRANSACTION`` statement for Postgres compatibility,
+  but as CrateDB does not support transactions, this command has no effect.
```

Comment:
```suggestion
- CrateDB now accepts the ``START TRANSACTION`` statement for :ref:`PostgreSQL
  wire protocol <postgres_wire_protocol>` compatibility. However, CrateDB does
  not support transactions and will silently ignore this statement.
```

changes:

- reworded it slightly so we say CrateDB merely "accepts" the statement (as opposed to "supports") because "supports" may imply something more than a no-op. additionally, we use the word "support" later in this bullet point to say CrateDB _doesn't_ support transactions. reserving "support" for "this feature is implemented and is functional" probably improves clarity/precision
- expanded "Postgres" to "PostgreSQL" because we should use product names in full
- expanded "PostgresSQL" to "PostgreSQL wire protocol" and added a link to the respective page in the documentation (its usually good to cross-reference other parts of the docs when possible, even from the release notes)
- chopped the sentence into two to simplify reading. reworded the second sentence to make CrateDB more of an active participant